### ISSUE TITLE
Postgis build failure with 1.3.3

### ISSUE THREAD

**Issue by lbartoletti** [Oslandia/SFCGAL#160]
I'm gonna run some more tests
But actually, postgis failed with this error:
```
gmake[4]: Entering directory '/wrkdirs/usr/ports/databases/postgis23/work/postgis-2.3.7/raster/loader'
cc -O2 -pipe  -I/usr/local/include -fstack-protector -fno-strict-aliasing  -fPIC -DPIC  -Wall -Wmissing-prototypes -I../rt_core -I../../liblwgeom  -I/usr/local/include -I/usr/local/include  -I/usr/local/include -c raster2pgsql.c
/bin/sh ../../libtool --mode=link cc -O2 -pipe  -I/usr/local/include -fstack-protector -fno-strict-aliasing  -fPIC -DPIC  -Wall -Wmissing-prototypes -I../rt_core -I../../liblwgeom  -I/usr/local/include -I/usr/local/include  -I/usr/local/include ../rt_core/librtcore.a raster2pgsql.o -L/usr/local/lib -L/usr/local/lib -fstack-protector ../../liblwgeom/liblwgeom.la -L/usr/local/lib -lgdal  -L/usr/local/lib -lgeos_c -lintl -L/usr/local/lib -liconv -lm -o raster2pgsql
libtool: link: cc -O2 -pipe -I/usr/local/include -fstack-protector -fno-strict-aliasing -fPIC -DPIC -Wall -Wmissing-prototypes -I../rt_core -I../../liblwgeom -I/usr/local/include -I/usr/local/include -I/usr/local/include raster2pgsql.o -fstack-protector -o .libs/raster2pgsql  ../rt_core/librtcore.a -L/usr/local/lib ../../liblwgeom/.libs/liblwgeom.so -lgdal -lgeos_c -lintl -liconv -lm -Wl,-rpath -Wl,/usr/local/lib
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::is_binary(std::__1::basic_ios<char, std::__1::char_traits<char> >&)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::BigFloatRep::operator new(unsigned int)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::BigFloatRep::toDouble() const'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::File_writer_OFF::write_header(std::__1::basic_ostream<char, std::__1::char_traits<char> >&, unsigned int, unsigned int, unsigned int, bool)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::BigIntRep::operator new(unsigned int)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::Random::Random(unsigned int)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::ORIGIN'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::postcondition_fail(char const*, char const*, int, char const*)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::get_mode(std::__1::basic_ios<char, std::__1::char_traits<char> >&)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::BigIntRep::operator delete(void*, unsigned int)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::BigFloatRep::operator delete(void*, unsigned int)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::assertion_fail(char const*, char const*, int, char const*)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::Random::Random()'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::precondition_fail(char const*, char const*, int, char const*)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::NULL_VECTOR'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::is_pretty(std::__1::basic_ios<char, std::__1::char_traits<char> >&)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::BigFloatRep::sqrt(CORE::BigFloatRep const&, CORE::extLong const&)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::warning_fail(char const*, char const*, int, char const*)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CGAL::File_header_OFF::File_header_OFF(bool, bool, bool, bool)'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::defBFsqrtAbsPrec'
/usr/local/lib/libSFCGAL.so.1: undefined reference to `CORE::BigFloatRep::BigFloatRep(double)'
cc: error: linker command failed with exit code 1 (use -v to see invocation)
gmake[4]: *** [Makefile:85: raster2pgsql] Error 1
gmake[4]: Leaving directory '/wrkdirs/usr/ports/databases/postgis23/work/postgis-2.3.7/raster/loader'
gmake[3]: *** [Makefile:31: rtloader] Error 2
gmake[3]: Leaving directory '/wrkdirs/usr/ports/databases/postgis23/work/postgis-2.3.7/raster'
gmake[2]: *** [GNUmakefile:16: all] Error 1
gmake[2]: Leaving directory '/wrkdirs/usr/ports/databases/postgis23/work/postgis-2.3.7'
gmake[1]: *** [Makefile:6: all] Error 2
gmake[1]: Leaving directory '/wrkdirs/usr/ports/databases/postgis23/work/postgis-2.3.7'
===> Compilation failed unexpectedly.
```

**Comment by lbartoletti** [comment]
@robe2 just tested SFCGAL 1.3.3 build also with CGAL 4.10 but postgis 23 reports the same errors.

FYI, SFCGAL is a default option for postgis23 and postgis24 on FreeBSD

**Comment by vmora** [comment]
fixed in 1.3.4.

**Comment by lbartoletti** [comment]
Thank you @vmora
